vue-frontend/src/components/jobs/LifeAtCanopas.vue

```diff
@@ -1,17 +1,19 @@
 <template>
-  <div class="pb-32">
-    <div class="life-container normal-text text-center">
+  <div class="life-at-canopas">
```

Comment:
Will this class be used? If yes, let's add tailwind CSS for the same

Reply:
No this class is not used any were. this class just for identification this is life at canopas section

Reply:
Let's remove it, as we are not adding these type of class anywhere

Reply:
okay